core-bundle/src/DependencyInjection/Compiler/RegisterFragmentsPass.php

```diff
@@ -160,7 +160,11 @@ private function registerFragment(ContainerBuilder $container, Reference $refere
         }
 
         $registry->addMethodCall('add', [$identifier, $config]);
-        $definition->addTag($tag, $attributes);
+
+        // Remove all the arrays from tag attribute as they are not supported
+        $definition->addTag($tag, array_filter($attributes, function ($v) {
```

Comment:
Should be is_scalar() instead. I guess an object is not supported either :)

Reply:
Good point, fixed.